src/uniprotkb/components/protein-data-views/__tests__/__mocks__/diseaseInvolvementUIData.ts

```diff
@@ -1,85 +1,192 @@
 import { DiseaseComment } from '../../../../types/commentTypes';
+import { FeatureDatum } from '../../UniProtKBFeaturesView';
 
-const mock: DiseaseComment[] = [
-  {
-    commentType: 'DISEASE',
-    disease: {
-      diseaseId: 'Alzheimers',
-      diseaseAccession: 'ASDASDAS',
-      acronym: 'AD',
-      description: 'Some description',
-      diseaseCrossReference: {
-        database: 'EMBL',
-        id: 'some ref id',
-        properties: {
-          prop1: 'value1',
-        },
-        isoformId: 'some isoform',
+const mock: { comments: DiseaseComment[]; features: FeatureDatum[] } = {
```

Comment:
Is this from an endpoint? Would it be possible to list the meta info like we do in other mocks eg:

```
// Source: /api/uniprotkb/search?facets=reviewed%2Cmodel_organism%2Cproteins_with%2Cexistence%2Cannotation_score%2Clength&query=glucose&size=25
// Retrieved: 2022-08-02
```

Reply:
That's not from an endpoint, I just modified the pre-PR mock (that looks like it also wasn't from an endpoint) in order to cover more edge-cases.